sklearn/linear_model/_coordinate_descent.py

```diff
@@ -1008,7 +972,7 @@ def fit(self, X, y, sample_weight=None, check_input=True):
             y,
             None,
             self.precompute,
-            _normalize,
+            False,
```

Comment:
Can we make this `normalize=False`? Would make it easier to know what this "lonely boolean" is about and help with grep'ing for instances of `normalize`?